akka-docs/src/main/paradox/stream/operators/Sink/cancelled.md

```diff
@@ -13,6 +13,16 @@ Immediately cancel the stream
 
 Immediately cancel the stream
 
+## Example
+
+In this example, we have a source that generates numbers from 1 to 5 but as we have used cancelled we get NotUsed as materialized value and stream cancels.
```

Comment:
```suggestion
In this example, we have a source that generates numbers from 1 to 5 but as we have used cancelled we get `NotUsed` as materialized value and stream cancels.
```

Reply:
I think docs play an important role for learners and so I'm contributing... I'll commit this suggestion then you can merge it